projects/controllers/tasks/tasks.py

```diff
@@ -210,6 +208,13 @@ def create_task(self, task: schemas.TaskCreate):
                 task.experiment_notebook = EXPERIMENT_NOTEBOOK
                 task.deployment_notebook = DEPLOYMENT_NOTEBOOK
 
+        # Adding the task name if it is a copy.
+        if task.copy_from:
+            task.name = self.generate_name_task(stored_task.name + " - Cópia")
```

Comment:
@dnlcesilva que tal trocar + por f-string: `f"{stored_task.name} - Cópia"`. 
A vantagem da f-string é relacionada a desempenho e facilidade de leitura.:
https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

Neste caso não faz diferença, já que é uma string pequena, mas é preferível utilizá-la sempre que possível.